File: common/src/main/java/com/thornbirds/component/view/IOrientationListener.java
package com.thornbirds.component.view;

/**
 * Basic interface for Orientation.
 *
 * @author YangLi yanglijd@gmail.com
 */
public interface IOrientationListener {
    /**
     * Called when screen orientation changes.
     *
     * @param isLandscape is landscape
     */
    void onOrientation(boolean isLandscape);
}

File: common/src/main/java/com/thornbirds/component/view/IOrientationListener.kt
package com.thornbirds.component.view

/**
 * Basic interface for Orientation.
 *
 * @author YangLi yanglijd@gmail.com
 */
interface IOrientationListener {
    /**
     * Called when screen orientation changes.
     *
     * @param isLandscape is landscape
     */
    fun onOrientation(isLandscape: Boolean)
}